### ISSUE TITLE
Using a new NPM dependency `set-harmonic-interval` breaks in IE11

### ISSUE THREAD

**Issue by eugenet8k** [streamich/react-use#575]
The recently added by @streamich dependency https://github.com/streamich/set-harmonic-interval does not seem to be processed by Babel. There is an arrow function `=>` in the distributed JS file hence it breaks in IE11.

**A little about versions:**  
- _OS_: Windows 10
- _Browser (vendor and version)_: IE11
- _`react-use`_:  11.3.2
- _Did this worked in the previous package version?_  Yes, it worked in v10.5.0

**Comment by andreiho** [comment]
Here's a workaround with Babel:

```
{
  test: /\.js$/,
  exclude: [new RegExp(`node_modules\/(?!(set-harmonic-interval)\/).*`)],
  loader: 'babel-loader',
  options: {
    presets: ['@babel/preset-env'],
  },
},
```

But of course it would be nice if the author did this themselves. @streamich

**Comment by psachs21** [comment]
Also, there are other issues with IE11. useLocation breaks in IE11 because `Event` cannot be called so the attempt to patch the history method just breaks.

**Comment by leepowellcouk** [comment]
PR has been put forward (https://github.com/streamich/set-harmonic-interval/pull/5) hopefully it will get merged.

**Comment by syabro** [comment]
actually it's https://github.com/streamich/set-harmonic-interval/pull/4  :)

**Comment by syabro** [comment]
if you don't use `useHarmonicInterval` then using `babel-plugin-import` as being told in https://github.com/streamich/react-use/blob/master/docs/Usage.md solves the IE11 issue

**Comment by xobotyi** [comment]
Closing, since [PR](https://github.com/streamich/set-harmonic-interval/pull/4) is merged and released under v1.0.1
